alert-policies/flutter-android/AppCrashRate.yml

```diff
@@ -0,0 +1,41 @@
+# Name of the alert
+name: App Crash Rate
+
+# Description and details
+description: |+
+  This alert is triggered when the application crash rate is above 20% for at least 5 mins.
+# Type of alert
+type: STATIC
+
+# NRQL query
+nrql:
+
+  query: "SELECT percentage(uniqueCount(sessionId), where category LIKE 'Crash') as 'Crash rate' FROM MobileSession, MobileCrash"
+
+# Function used to aggregate the NRQL query value(s) for comparison to the terms.threshold (Default: SINGLE_VALUE)
+valueFunction: SINGLE_VALUE
+
+# List of Critical and Warning thresholds for the condition
+terms:
+  - priority: CRITICAL
+    # Operator used to compare against the threshold.
+    operator: ABOVE
+    # Value that triggers a violation
+    threshold: 0.2
```

Comment:
Here, the value received will be in percentage, will this threshold work as per the description ?

Reply:
Yes, It works when the error rate is above 20%  means (0.2) got the alert.